ajax/libs/jquery-ui-bootstrap/package.json

```diff
@@ -7,12 +7,22 @@
     "jquery",
     "bootstrap"
   ],
-  "license": {
-    "name": "MIT License",
-    "url": "http://www.opensource.org/licenses/mit-license.php"
-  },
   "repository": {
     "type": "git",
     "url": "https://github.com/jquery-ui-bootstrap/jquery-ui-bootstrap.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jquery-ui-bootstrap/jquery-ui-bootstrap.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "css/custom-theme/jquery-ui*",
+          "third-party/**/*"
```

Comment:
It needs the folder `assets`, because the origin `filename` is `assets/css/bootstrap.min.css`. ("filename": "assets/css/bootstrap.min.css")
Maybe we should add `assets/**/*` .